CMakeLists.txt

```diff
@@ -0,0 +1,610 @@
+        src/moaicore/MOAIVertexFormatMgr.cpp
+        src/moaicore/MOAIViewport.cpp
+        src/moaicore/MOAIWheelSensor.cpp
+        src/moaicore/MOAIXmlParser.cpp
+        src/moaicore/MOAIXmlWriter.cpp
+        src/uslscore/STLString.cpp
+        src/uslscore/USAdapterInfo_posix.cpp
+        src/uslscore/USAdapterInfo_win32.cpp
+        src/uslscore/USBase64Encoder.cpp
+        src/uslscore/USBase64Reader.cpp
+        src/uslscore/USBase64Writer.cpp
+        src/uslscore/USBox.cpp
+        src/uslscore/USByteStream.cpp
+        src/uslscore/USCgt.cpp
+        src/uslscore/USColor.cpp
+        src/uslscore/USCurve.cpp
+        src/uslscore/USDeflateReader.cpp
+        src/uslscore/USDeflateWriter.cpp
+        src/uslscore/USDeviceTime_apple.cpp
+        src/uslscore/USDeviceTime_nacl.cpp
+        src/uslscore/USDeviceTime_posix.cpp
+        src/uslscore/USDeviceTime_win32.cpp
+        src/uslscore/USDirectoryItr.cpp
+        src/uslscore/USDistance.cpp
+        src/uslscore/USFileStream.cpp
+        src/uslscore/USFileSys.cpp
+        src/uslscore/USFrustum.cpp
+        src/uslscore/USHashWriter.cpp
+        src/uslscore/USHashWriterAdler32.cpp
+        src/uslscore/USHashWriterCRC32.cpp
+        src/uslscore/USHashWriterMD5.cpp
+        src/uslscore/USHashWriterSHA1.cpp
+        src/uslscore/USHashWriterSHA224.cpp
+        src/uslscore/USHashWriterSHA256.cpp
+        src/uslscore/USHashWriterSHA384.cpp
+        src/uslscore/USHashWriterSHA512.cpp
+        src/uslscore/USHashWriterWhirlpool.cpp
+        src/uslscore/USHexDump.cpp
+        src/uslscore/USHexReader.cpp
+        src/uslscore/USHexWriter.cpp
+        src/uslscore/USInterpolate.cpp
+        src/uslscore/USIntersect.cpp
+        src/uslscore/USLexStream.cpp
+        src/uslscore/USLog.cpp
+        src/uslscore/uslscore-pch.cpp
+        src/uslscore/USMemStream.cpp
+        src/uslscore/USMercator.cpp
+        src/uslscore/USParser.cpp
+        src/uslscore/USPlane.cpp
+        src/uslscore/USPolar.cpp
+        src/uslscore/USPrism.cpp
+        src/uslscore/USQuad.cpp
+        src/uslscore/USQuadCoord.cpp
+        src/uslscore/USQuaternion.cpp
+        src/uslscore/USRhombus.cpp
+        src/uslscore/USStream.cpp
+        src/uslscore/USStreamReader.cpp
+        src/uslscore/USStreamWriter.cpp
+        src/uslscore/USSurface2D.cpp
+        src/uslscore/USSyntaxNode.cpp
+        src/uslscore/USSyntaxScanner.cpp
+        src/uslscore/USTrig.cpp
+        src/uslscore/USTypedPtr.cpp
+        src/uslscore/USXmlReader.cpp
+        src/uslscore/USZip.cpp
+        src/uslscore/USZipFile.cpp
+        src/zlcore/zlcore-pch.cpp
+        src/zlcore/zl_mutex.cpp
+        src/zlcore/zl_util.cpp
+        src/zlcore/zl_vfscanf.cpp
+        src/zlcore/zlcore.cpp
+        src/zlcore/ZLDirectoryItr.cpp
+        src/zlcore/ZLFile.cpp
+        src/zlcore/ZLFileSystem.cpp
+        src/zlcore/ZLVirtualPath.cpp
+        src/zlcore/ZLZipArchive.cpp
+        src/zlcore/ZLZipStream.cpp)
```

Comment:
So we could potentially gather all files that need to be compiled without having to add them all, but this is the recommended way to do it and we don't touch moai much/at all. If we ever added new sources to moai we would have to add them here so they are available in android.